packages/ui/src/components/StarburstLegend.tsx

```diff
@@ -97,7 +98,23 @@ export interface StarburstLegendProps {
   backgroundColor: string;
 }
 
+function generateIntervalSequence(
+  config: SpacedRepetitionSchedulerConfiguration,
+  maxIntervals: number,
+) {
+  const sequence = [{ interval: config.initialReviewInterval, label: "tmp" }];
+  for (let i = 0; i < maxIntervals; i++) {
+    sequence.push({
+      interval:
+        sequence[sequence.length - 1].interval * config.intervalGrowthFactor,
+      label: "tmp",
+    });
+  }
+  return sequence;
+}
+
```

Comment:
Right! I think the right way to think about it is that this is now a `ui` concern: at least for the moment, it's really just presentational. But I did find that it's best to massage the labels a little bit, so that they aren't ugly "18 days" or whatever: I think we probably do still want to hard-code a literal sequence so that we can have "nice" labels as before.

Reply:
Gotcha! I just implemented a fixed interval sequence in the latest commit, let me know if that's what you had in mind!